media-libs/avidemux-plugins/avidemux-plugins-2.6.20.ebuild

```diff
@@ -0,0 +1,164 @@
+	~media-video/avidemux-${PV}:${SLOT}[opengl?,qt4?,qt5?]
+	>=dev-lang/spidermonkey-1.5-r2:0=
+	dev-libs/libxml2:2
+	media-libs/libpng:0=
+	virtual/libiconv:0
+	aac? (
+		media-libs/faac:0
+		media-libs/faad2:0
+	)
+	aften? ( media-libs/aften:0 )
+	alsa? ( >=media-libs/alsa-lib-1.0.3b-r2:0 )
+	amr? ( media-libs/opencore-amr:0 )
+	dts? ( media-libs/libdca:0 )
+	fontconfig? ( media-libs/fontconfig:1.0 )
+	fribidi? ( dev-libs/fribidi:0 )
+	jack? (
+		media-sound/jack-audio-connection-kit:0
+		libsamplerate? ( media-libs/libsamplerate:0 )
+	)
+	lame? ( media-sound/lame:0 )
+	nvenc? ( amd64? ( media-video/nvidia_video_sdk:0 ) )
+	opus? ( media-libs/opus:0 )
+	pulseaudio? ( media-sound/pulseaudio:0 )
+	truetype? ( media-libs/freetype:2 )
+	twolame? ( media-sound/twolame:0 )
+	x264? ( media-libs/x264:0= )
+	x265? ( media-libs/x265:0= )
+	xv? (
+		x11-libs/libX11:0
+		x11-libs/libXext:0
+		x11-libs/libXv:0
+	)
+	xvid? ( media-libs/xvid:0 )
+	vorbis? ( media-libs/libvorbis:0 )
+	vpx? ( media-libs/libvpx:0 )
+	system-a52dec? ( media-libs/a52dec:0 )
+	system-libass? ( media-libs/libass:0= )
+	system-libmad? ( media-libs/libmad:0 )
+	system-libmp4v2? ( media-libs/libmp4v2:0 )
+	${PYTHON_DEPS}
+"
+DEPEND="${RDEPEND}
+	oss? ( virtual/os-headers:0 )"
+
+S="${WORKDIR}/${MY_P}"
+
+PATCHES=( "${FILESDIR}"/${PN}-2.6.20-optional-pulse.patch )
+
+src_prepare() {
+	# Don't call cmake-utils_src_prepare since there is no CMakeLists.txt in ${S}
+	default
+}
+
+src_configure() {
+	# Add lax vector typing for PowerPC.
+	if use ppc || use ppc64 ; then
+		append-cflags -flax-vector-conversions
+	fi
+
+	# See bug 432322.
+	use x86 && replace-flags -O0 -O1
+
+	processes="buildPluginsCommon:avidemux_plugins
+		buildPluginsCLI:avidemux_plugins"
+	use qt4 && processes+=" buildPluginsQt4:avidemux_plugins"
+
+	for process in ${processes} ; do
+		local build="${WORKDIR}/${P}_build/${process%%:*}"
+
+		local mycmakeargs=(
+			-DAVIDEMUX_SOURCE_DIR='${S}'
+			-DPLUGIN_UI=$(echo ${build/buildPlugins/} | tr '[:lower:]' '[:upper:]')
+			-DFAAC="$(usex aac)"
+			-DFAAD="$(usex aac)"
+			-DALSA="$(usex alsa)"
+			-DAFTEN="$(usex aften)"
+			-DOPENCORE_AMRWB="$(usex amr)"
+			-DOPENCORE_AMRNB="$(usex amr)"
+			-DLIBDCA="$(usex dts)"
+			-DFONTCONFIG="$(usex fontconfig)"
+			-DJACK="$(usex jack)"
+			-DLAME="$(usex lame)"
+			-DNVENC="$(usex nvenc)"
+			-DOPUS="$(usex opus)"
+			-DOSS="$(usex oss)"
+			-DPULSEAUDIOSIMPLE="$(usex pulseaudio)"
+			-DQT4="$(usex qt4)"
+			-DFREETYPE2="$(usex truetype)"
+			-DTWOLAME="$(usex twolame)"
+			-DX264="$(usex x264)"
+			-DX265="$(usex x265)"
+			-DXVIDEO="$(usex xv)"
+			-DXVID="$(usex xvid)"
+			-DVDPAU="$(usex vdpau)"
+			-DVORBIS="$(usex vorbis)"
+			-DLIBVORBIS="$(usex vorbis)"
+			-DVPXDEC="$(usex vpx)"
+			-DUSE_EXTERNAL_LIBA52="$(usex system-a52dec)"
+			-DUSE_EXTERNAL_LIBASS="$(usex system-libass)"
+			-DUSE_EXTERNAL_LIBMAD="$(usex system-libmad)"
+			-DUSE_EXTERNAL_LIBMP4V2="$(usex system-libmp4v2)"
```

Comment:
What happens if you disable those ? Does it build a bundled one ?

If so, please make it mandatory, we don't want to track all of its bundled libs for security issues